tests/test_sanitise_text.py

```diff
@@ -37,14 +36,16 @@ def test_encode_chars_the_same_for_ascii_and_gsm(char, expected, cls):
     (('\n', '\n', '?'), 'ascii control char in gsm (newline)'),
     (('\r', '\r', '?'), 'ascii control char in gsm (return)'),
     # These characters are present in GSM but not in ascii
+    (('â', 'â', 'a'), 'non-ascii gsm char (a with hat)'),
     (('à', 'à', 'a'), 'non-ascii gsm char (a with accent)'),
+    (('Ŷ', 'Ŷ', 'Y'), 'non-ascii gsm char (capital y with hat)'),
```

Comment:
can you move these two added lines to the bottom and add a separate comment to say that they're not GSM but are welsh characters